wire/internal/wire/testdata/UnusedInputs/foo/foo.go

```diff
@@ -0,0 +1,54 @@
+// Copyright 2018 Google LLC
```

Comment:
(For the directory, not this particular file/line:) Wire uses the term "input" to mean arguments passed to the injector. A better name for this test would be `UnusedProviders`.

Reply:
Done.